Packs/AzureDataExplorer/Integrations/AzureDataExplorer/AzureDataExplorer.yml

```diff
@@ -0,0 +1,487 @@
+category: Analytics & SIEM
+commonfields:
+  id: AzureDataExplorer
+  version: -1
+configuration:
+- defaultvalue: https://help.kusto.windows.net
+  display: Cluster URL (e.g. https://help.kusto.windows.net)
+  name: cluster_url
+  required: true
+  type: 0
+- display: Application ID
```

Comment:
we should provide a default app ID (as described on Slack)

Reply:
Done.